### PR TITLE
Update Kotlin version for all exercises

### LINKED ISSUE #288: Upgrade to the latest Kotlin

### PR DESCRIPTION
**Pull request by stkent**
Depends on ~~#316~~ #323 for CI fixes, do not merge before that.

Fixes #288 (for now!)

### CODE REVIEW

File: exercises/reverse-string/build.gradle [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 test {
-    testLogging {
-        exceptionFormat = 'full'
-        events = ["passed", "failed", "skipped"]
-    }
+  testLogging {
+    exceptionFormat = 'full'
+    events = ["passed", "failed", "skipped"]
+  }
 }
```
Comment (stkent) [inline exercises/reverse-string/build.gradle]: Normalized indentation to match all other build.gradle files.

**Comment by mikegehard** [comment]
Now depends on https://github.com/exercism/kotlin/pull/323

**Comment by mikegehard** [comment]
would it also be good to update the Kotlin version in https://github.com/exercism/kotlin/blob/master/_template/build.gradle as well?

**Comment by stkent** [comment]
Good catch; done!

**Pull request merged**